packages/jellyfish-api-core/__tests__/category/loan/setCollateralToken.test.ts

```diff
@@ -63,6 +63,24 @@ describe('Loan setCollateralToken', () => {
     await expect(promise).rejects.toThrow('RpcApiError: \'Amount out of range\', code: -3, method: setcollateraltoken')
   })
 
+  it('should not setCollateralToken if oracle does not contain USD price', async () => {
+    await testing.token.create({ symbol: 'TSLA' })
+    await testing.generate(1)
+
+    const priceFeedId: string = await testing.container.call('appointoracle', [await testing.generateAddress(), [{
+      token: 'TSLA',
+      currency: 'SGD'
+    }], 1])
+    await testing.generate(1)
+
+    const promise = testing.rpc.loan.setCollateralToken({
+      token: 'TSLA',
+      factor: new BigNumber(0.5),
+      priceFeedId
+    })
+    await expect(promise).rejects.toThrow(`RpcApiError: 'Test LoanSetCollateralTokenTx execution failed:\noracle (${priceFeedId}) does not conntain USD price for this token!', code: -32600, method: setcollateraltoken`)
```

Comment:
Reviewers, please ignore the typo `conntain`, I shall inform C++ programmer to fix this